src/libraries/Directory.Build.targets

```diff
@@ -219,6 +219,10 @@
   <Target Name="GetDocumentationFile"
           Returns="$(DocumentationFile)"/>
 
+  <PropertyGroup>
+    <DefineConstants Condition="'$(IsPrerelease)' != 'false' and '$(TargetFramework)' != '$(NetCoreAppCurrent)'">$(DefineConstants),USE_INTERNAL_ACCESSIBILITY</DefineConstants>
```

Comment:
I'm not sure I like this living in Directory.Build.props. Shouldn't it rather be something that individual projects opt into?

Reply:
That was my first reaction too. Levi suggested that it live here, so I didn't pay too much attention to it.  In my next PR, I'll move it to System.Utf8Experimental.csproj.

Reply:
Is this constant meant to be used in contexts other than intrinsics shims? If not, then we should probably use a more specific name like `USE_INTERNAL_INTRINSICS_SHIMS` or something.

Reply:
From an infrastructure standpoint this is wrong as the `TargetFramework` property shouldn't be read outside the project file. Instead inferred properties like `TargetFrameworkVersion` and `TargetFrameworkIdentifier` should be used. If you already plan to move it then it's fine.